.CONTRIBUTING.md

```diff
@@ -66,7 +66,13 @@ To set up the structure, follow these steps:
 
     This command will install all dependencies listed in the `requirements.txt` file.
 
-2. In the `polkadot-mkdocs` folder (which should be the current one), you can build the site by running:
+2. Switch to the polkadot-docs directory and use [npm](https://docs.npmjs.com/about-npm) to install dependencies:
+
+    ```bash
+    cd polkadot-docs && npm install
+    ```
+
+3. In the `polkadot-mkdocs` folder, you can build the site by running:
 
     ```bash
     mkdocs serve
```

Comment:
```suggestion
    cd ..
    mkdocs serve
```

Reply:
Let's get them back to the right folder safely

Reply:
Yeah, I considered adding `cd ..`, but I'm a bit unsure about it.

I was worried about misleading users into thinking they can serve the site only if they run `cd ..` first (which might seem silly, but if it's all together, it could lead to that). WDYT?